docs/source/sections/Installation.rst

```diff
@@ -6,27 +6,28 @@ In this section we will outline the installation steps for installing the softwa
 
 The first stage is installing a version of Python 3.9, if you don't have a working version. We suggest installing a working Anaconda distribution from https://www.anaconda.com/products/distribution#macos following the instructions on that page.
 
-Installing PolarRoute
+Installing MeshiPhi
 #####################
 
-The PolarRoute software can be installed on Windows/Linux/MacOS by running one of the two following commands.
+The MeshiPhi software can be installed on Windows/Linux/MacOS by running one of the two following commands.
 
-Github:
+
+Pip: 
 ::
 
-    git clone https://https://github.com/antarctica/PolarRoute.git
-    python setup.py install
+    pip install MeshiPhi
 
-Pip: 
+Github:
 ::
 
-    pip install polar-route
+    git clone https://https://github.com/antarctica/MeshiPhi.git
+    python setup.py install
```

Comment:
Running this in a fresh venv with a freshly cloned copy of MeshiPhi gives `ModuleNotFoundError: No module named 'numpy'`. Replace with `pip install ./MeshiPhi` and it works.

Reply:
This was on ubuntu, possibly worth testing on other platforms. But `python setup.py install` is now deprecated: https://packaging.python.org/en/latest/discussions/setup-py-deprecated/